test.lua

```diff
@@ -298,7 +300,7 @@ test:test('simple expires test',  function(test)
     -- put test tuples
     log.info("-------------- put ------------")
     log.info("put to space " .. prefix_space_id(space_id))
-    put_test_tuples(space_id, 10)
+    put_test_tuples(space_id, 10, 120)
```

Comment:
So we expect 5 tuples to expire? I think it's worth adding a comment here about this. It's not obvious what should happen after one calls `put_test_tuples(space_id, 10, 120)` until one scroll a couple hundred lines above.

Reply:
And I don't quite get why below we assert `return cnt == 7`

Reply:
> And I don't quite get why below we assert `return cnt == 7`

Because `put_test_tuples` adds 2 tuples that always expire (field_no == 3 != number):
```
157     -- tuple w/o expiration date                                                
158     local uid = total + 1                                                       
159     add_entry(space_id, uid, get_email(uid), "")                                
160                                                                                 
161     -- tuple w/ invalid expiration date                                         
162     uid = total + 2                                                             
163     add_entry(space_id, uid, get_email(uid), "some string in exp field")
```

Reply:
Ok, this should be enough